docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
     image: rabbitmq:latest
 
   elasticsearch:
-    image: elasticsearch:5
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.2.4
```

Comment:
this might have to be `docker.elastic.co/elasticsearch/elasticsearch-oss:6.2.4` otherwise `x-pack` would get installed which can interfere with the tests a bit. Just if you run into unexpected issues.

Reply:
It didn't affect the tests locally, but this is good to know. 👍